src/index.js

```diff
@@ -12,4 +13,5 @@ export {
   padLeft,
   randomInteger,
   arrayFill,
+  objectValuesToString
```

Comment:
I just saw you created you own config for eslint and that is was actually specified in the package.json! my mistake, I always work with an .eslintrc.json at the root of a project and, as I couldn't see any, I assumed I had to create one and use the default eslint:recommended.